modules/manage/pages/schema-reg/schema-reg-api.adoc

```diff
@@ -247,6 +212,15 @@ res = requests.get(f'{base_uri}/schemas/ids/1').json()
 pretty(res)
 ```
 
+--
+rpk::
++
+--
+
+```bash
+rpk registry schema get --id 1
```

Comment:
In rpk there are 3 ways to do this:

```
* By version, returning a schema for a required subject and version
  rpk registry schema get [subject] --schema-version <version>
  
* By ID, returning all subjects using the schema, or filtering for one subject
  rpk registry schema get --id <id>
* By schema, checking if the schema has been created in the subject
  rpk registry schema get [subject] --schema <path/to/file>
```

Maybe it's worth adding a note or linking to reference docs about this.

Reply:
@r-vasquez You should be able to see the rpk tabs as long as you set the docs version in the deploy preview to 23.3 Beta -- let me know if it doesn't work! 

For this particular example I just kept it consistent with the the curl and Python examples, which only retrieve the schema by ID (no subjects or versions). I also linked out to the Intro to rpk and rpk registry docs in the [Prerequisites](https://deploy-preview-106--redpanda-docs-preview.netlify.app/23.3/manage/schema-reg/schema-reg-api/#prerequisites) section. Does that work?